README.md

```diff
@@ -20,59 +20,60 @@
   </a>
 </p>
 
+*Spring Cloud Data Flow* is a microservices-based streaming and batch data processing pipelines in Cloud Foundry and
+Kubernetes.
 
-*Spring Cloud Data Flow* is a toolkit for building data integration and real-time data processing pipelines. 
-
-Pipelines consist of Spring Boot apps, built using the [Spring Cloud Stream](https://github.com/spring-cloud/spring-cloud-stream)
+Data processing pipelines consist of Spring Boot apps, built using the [Spring Cloud Stream](https://github.com/spring-cloud/spring-cloud-stream)
 or [Spring Cloud Task](https://github.com/spring-cloud/spring-cloud-task) microservice frameworks. 
 
-This makes Spring Cloud Data Flow suitable for a range of data processing use cases, from import/export to 
-event streaming and predictive analytics.
+This makes Spring Cloud Data Flow suitable for a range of data processing use cases, from import/export to event streaming
+and predictive analytics.
 
 ----
 
 ## Components
 
-The [Core](https://github.com/spring-cloud/spring-cloud-dataflow/tree/master/spring-cloud-dataflow-core)
-domain module includes the concept of a *stream* that is a composition of spring-cloud-stream
-modules in a linear pipeline from a *source* to a *sink*, optionally including *processor* application(s)
-in between. The domain also includes the concept of a *task*, which may be any process that does
-not run indefinitely, including [Spring Batch](https://github.com/spring-projects/spring-batch) jobs.
-
-The [App Registry](https://github.com/spring-cloud/spring-cloud-dataflow/tree/master/spring-cloud-dataflow-registry)
-maintains the set of available apps, and their mappings to URIs.
-For example, if relying on Maven coordinates, an app's URI would be of the format:
-`maven://<groupId>:<artifactId>:<version>`
+**Architecture**: The Data Flow Server is a Spring Boot application that provides REST APIs, UI, and Java DSL.
+A single Data Flow Server instance can support deploying streams and tasks to Local, Cloud Foundry, and Kubernetes.
```

Comment:
Actually we also need a database server and skipper server to support deploying streams.  Maybe - A single Data Flow *installation* can support...